### ISSUE TITLE
Exception while running migrations

### ISSUE THREAD

**Issue by erez-rabih** [weavejester/ragtime#96]
Hi,

I am getting the following error when running the migrate function:
```
Exception Conflict! Expected ragtime.repl$wrap_reporting$reify__719@33ee656f but ragtime.repl$wrap_reporting$reify__719@f3c1ebe was applied.  ragtime.strategy/raise-error (strategy.clj:39)
```

Anything I can do to solve this?

**Comment by danielcompton** [comment]
You might need to provide more information on your migrations. That error message by itself isn't enough to debug it.

**Comment by erez-rabih** [comment]
What would help investigating this?

**Comment by weavejester** [comment]
Actually, you've supplied enough information. You're running into a bug caused by an interaction between ragtime.repl, ragtime.core and ragtime.strategy. What should happen is that you're told the IDs of the migrations that conflict, but ragtime.core is missing a function call. I'll put in a fix.

**Comment by weavejester** [comment]
This should be fixed in 0.5.3. You should get a conflict message that tells you the IDs of the conflicting migrations.

**Comment by erez-rabih** [comment]
@weavejester Thanks for the fast response I appreciate it.
